### ISSUE TITLE
add information on different trigger levels

### ISSUE THREAD

**Issue by rmanzoni** [cms-tau-pog/TauTriggerValidation#3]
Just to note this down:
it would be useful to have the information on whether intermediate trigger levels (L1, L2, L2p5) are passed saved in the tuples.

**Comment by l-cadamuro** [comment]
Where can one easily get the filter names corresponding to these levels?

**Comment by rmanzoni** [comment]
Unfortunately there's no easy way other than going through the path and know what filters are relevant.

As a shorthand I usually save these
['hltL2IsoTau26eta2p2', 'hltL2Tau26eta2p2', 'hltL2TauIsoFilter', 'hltL2TauJetsIso'] # di-tau
['hltL2IsoTau26eta2p2', 'hltL2Tau26eta2p2', 'hltL2TauIsoFilter', 'hltL2TauJetsIso'] # tau + MET (includes L1)
['hltPFTau20', 'hltSelectedPFTausTrackFinding', 'hltPFTau20Track', 'hltSelectedPFTausTrackFindingLooseIsolation', 'hltPFTau20TrackLooseIso', 'hltPFTauAgainstMuonDiscriminator', 'hltSelectedPFTausTrackFindingLooseIsolationAgainstMuon', 'hltPFTau20TrackLooseIsoAgainstMuon', 'hltOverlapFilterSingleIsoMu19LooseIsoPFTau20', 'hltOverlapFilterSingleIsoMu21LooseIsoPFTau20'] # mu-tau single L1

but some of these, especially for mt, are either not available by default (saveTags = False) or redundant.

bottom line: for the development you can refer to the filters for di-tau, then we need to review the thing before deploying it as definitive

**Comment by l-cadamuro** [comment]
Ok, as long as one has the filter list, we can use the same logic to check other levels.
Then one expert will take care to keep the list up to date :-)
Then we simply need to remove the part where we check if the trigger was fired.
